addons/mrp/models/stock_move.py

```diff
@@ -242,6 +243,40 @@ def default_get(self, fields_list):
                 defaults['reference'] = production_id.name
         return defaults
 
+    @api.model_create_multi
+    def create(self, values):
+        """ Enforce consistent values (i.e. match _get_move_raw_values/_get_move_finished_values) for:
+        - Manually added components/byproducts specifically values we can't set via view with "default_"
+        - Moves from a copied MO
+        - Backorders
+        """
+        mo_id_to_mo = defaultdict(lambda: self.env['mrp.production'])
+        product_id_to_product = defaultdict(lambda: self.env['product.product'])
+        for value in values:
+            mo_id = value.get('raw_material_production_id', False) or value.get('production_id', False)
+            if mo_id:
+                mo = mo_id_to_mo[mo_id]
+                if not mo:
+                    mo = mo.browse(mo_id)
+                    mo_id_to_mo[mo_id] = mo
+                value['name'] = mo.name
+                value['origin'] = mo._get_origin()
+                value['group_id'] = mo.procurement_group_id.id
+                value['propagate_cancel'] = mo.propagate_cancel
+                if value.get('raw_material_production_id', False):
+                    product = product_id_to_product[value['product_id']]
+                    if not product:
+                        product = product.browse(value['product_id'])
+                    product_id_to_product[value['product_id']] = product
+                    value['location_dest_id'] = mo.production_location_id.id
+                    value['price_unit'] = product.standard_price
```

Comment:
is that correctly overwrite in mrp_account? Shouldn't use _get_price_unit instead?

Reply:
Method isn't available in mrp module (requires stock_account). `standard_price` is what is already being used in productions, so it's at least consistent?